#provide motor speeds in rpm, turn radius in meters, and drive speed in m/s
uint16 C_STOP=0
uint16 C_MOTFL=1
uint16 C_MOTFR=2
uint16 C_MOTBL=3
uint16 C_MOTBR=4
uint16 C_MOTS=5
uint16 C_LEFT=6
uint16 C_RIGHT=7
uint16 C_LR=8
uint16 C_DRV=9
uint16 C_TRN=10
uint16 C_DRVTRN=11

uint16 cmd
float32[] data
